rfcs/0038-rosetta-construction-api.md

```diff
@@ -269,7 +269,33 @@ This is a simple GraphQL query. This endpoint should be easy to implement.
 
 [via Payloads](#payloads)
 
-The Rosetta spec leaves the encoding of unsigned transactions implementation-defined. Since we'll default to using our client-sdk's signing mechanism our encoding will be precisely the JSON input (stringified) demanded by the "unsafe" method of the client-sdk API:
+The Rosetta spec leaves the encoding of unsigned transactions implementation-defined. Since we want to make it easy for alternate signers to be created (eg. the ledger), we'll want this encoding to be some faithful representation of the bytes upon which the signature operation acts.
+
+Specifically this is the user command having been transformed into a `Transaction_union_payload.t` and then hashed into a `(field, bool) Random_oracle_input.t`. We will serialzie the Random_oracle_input with a custom protocol as defined below and send that byte-buffer as hex-encoded ascii.
+
+```
+// Serialization schema for Random oracle input
```

Comment:
Specificlly, this is for the case for `(Fq.t,bool) Random_oracle_input.t`, yes?

Reply:
I'll keep it as "field" bool since both Fq and Fp are 32bytes so it doesn't make sense to distinguish them here